tests/benchmarks/README.md

```diff
@@ -0,0 +1,13 @@
+# Automated Benchmarking
+[Benchmark repo](https://github.com/rlworkgroup/benchmarks)
+
+We have an automated system that will run any benchmark file named "test\_benchmark_*.py" in this folder /tests/benchmarks/.
```

Comment:
why do you need to escape a backslash in markdown???

Reply:
I'm not sure, it would not show up in the preview otherwise.

Reply:
sorry i meant, escape underscore

Reply:
i realized the md previewer I was using required it, but github does not